tui/messages/adversarial_challenger_m3_1_test.go

```
@@ -0,0 +1,580 @@
+						cfg.desc, ar.desc, avail, bbox.Rows, effectiveMaxRows)
+				}
+
+				limitCols := avail
+				if cfg.maxCols > 0 && cfg.maxCols < limitCols {
+					limitCols = cfg.maxCols
+				}
+				if limitCols < 1 {
+					limitCols = 1
+				}
+				if limitCols > effectiveMaxCols {
+					limitCols = effectiveMaxCols
```

golangci-lint (2.13.2)

[error] ineffectual assignment to limitCols

(ineffassign)